app/KeywordExample.js

```diff
@@ -5,6 +5,7 @@ import styled from "styled-components";
 // Internal dependencies.
 import KeywordInput from "../composites/Plugin/Shared/components/KeywordInput";
 import SynonymsSection from "../composites/Plugin/Synonyms/components/SynonymsSection";
+import CornerstoneToggle from "../composites/Plugin/CornerstoneContent/CornerstoneToggle";
```

Comment:
this path should be "../composites/Plugin/CornerstoneContent/components/CornerstoneToggle"